src/Http/ServerRequestFactory.php

```diff
@@ -79,6 +81,7 @@ public static function fromGlobals(
             'base' => $uri->base,
             'session' => $session,
             'mergeFilesAsObjects' => Configure::read('App.uploadedFilesAsObjects', true),
+            'input' => $input,
```

Comment:
This create merge conflict with `4.next` since this is how passing input stream data is handled there
https://github.com/cakephp/cakephp/blob/b1436bf69b40cda9ec7bfcdae2a68406e7d7d998/src/Http/ServerRequestFactory.php#L82